frigate/embeddings/maintainer.py

```diff
@@ -236,6 +237,41 @@ def _process_updates(self) -> None:
 
             self.tracked_events[data["id"]].append(data)
 
+        # check if we're configured to send an early request after a minimum number of updates received
+        # we don't check required zones here - probably should, but does that run the risk in this simple logic
```

Comment:
I think that not checking required zones will confuse users unless we document it. Under what condition would we *not* want to check the required zones?

Reply:
My concern was that the "significant updates" counter might not make sense if it needed to be per-zone. Having looked through the way updates are generated though I think this is fine. I've added in the zones check.